L/libaom/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libaom"
+version = v"3.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://storage.googleapis.com/aom-releases/libaom-$(version).tar.gz", "bd754b58c3fa69f3ffd29da77de591bd9c26970e3b18537951336d6c0252e354")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libaom-*
+mkdir build-dir
+cd build-dir
+CMAKE_ARGS="-DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DENABLE_TESTS=false -DBUILD_SHARED_LIBS=1"
+if [[ "${target}" != "*86*" ]]; then
+    CMAKE_ARGS="${CMAKE_ARGS} -DAOM_TARGET_CPU=generic"
+fi
+cmake ${CMAKE_ARGS} ..
+make
+make install
+install_license ../LICENSE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+platforms = expand_cxxstring_abis(platforms)
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("aomenc", :aomenc),
+    ExecutableProduct("aomdec", :aomdec),
+    LibraryProduct(["libaom", "aom"], :libaom),
+]
+
+# Dependencies that must be installed before this package can be built
+#
+# YASM is recommended in the build instructions, but errors on apple platforms.
+# Assembly only exists for x86 targets.
+dependencies = [
+                HostBuildDependency("YASM_jll"; platforms=filter(p->occursin("86", triplet(p)) && !occursin("apple", triplet(p)), platforms)),
+                HostBuildDependency("NASM_jll"; platforms=filter(p->occursin("86", triplet(p)) && occursin("apple", triplet(p)), platforms)),
```

Comment:
This is a bit easier to read:
```suggestion
    HostBuildDependency("YASM_jll"; platforms=filter(p->proc_family(p) == "intel" && !Sys.isapple(p), platforms)),
    HostBuildDependency("NASM_jll"; platforms=filter(p->proc_family(p) == "intel" && Sys.isapple(p), platforms)),
```